python/tk_framework_desktopserver/server_protocol.py

```diff
@@ -26,6 +23,8 @@
 from autobahn.twisted.websocket import WebSocketServerProtocol
 from twisted.internet import error, reactor
 
+logger = get_logger(__name__)
```

Comment:
At a glance, some of this will step on things. You'll either have to manually merge this file, or do a rebase -i and get your changes on top of the newer code in zero_config.